types/btctypes/utxo.go

```diff
@@ -192,6 +193,28 @@ func (op outPoint) String() string {
 	return fmt.Sprintf("%s:%d", op.txHash, op.vout)
 }
 
```

Comment:
Can you add Write(w io.Writer) as it would be more clean, (as we already have a ReadOutPoint fn)

Reply:
A `Write(w io.Writer)` already exists on this type. The intention of this PR was to make `outpoint` implement the `json.Marshaler` and `json.UnMarshaler` interfaces. I am happy to implement this in the external application (that needs it) if you'd like.